tests/commands/test_build_config.py

```diff
@@ -90,16 +90,13 @@ def test_ask_user_overwrite(mocker):
     """
     Once https://github.com/tmbo/questionary/issues/35 is implemented, improve this test.
     """
-    prompt_mock = mocker.patch(
-        "freqtrade.commands.build_config_commands.prompt", return_value={"overwrite": False}
-    )
+    prompt_mock = mocker.patch("questionary.prompt", return_value={"overwrite": False})
```

Comment:
I think rather than patching into 3rd party modules (a code-smell for sure), moving the questionary stuff into a different file would be the better option (not here - but in the actual code file, obviously).

you can then import the function you call within the function - but import questionary "in the open" - so it's patchable easily.

Reply:
I agree with you, just make the tests pass and listen to your idea, my first thought was as what you said.